frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -50,11 +63,24 @@ const LineAreaBarChart = ({
     goalColor: getColor("text-medium"),
   };
 
+  const palette = { ...colors, ...instanceColors };
+  const mainSettings = multipleSettings[0];
```

Comment:
[Once we decided to make the legend consistent for multiple series dashcards](https://metaboat.slack.com/archives/C01LQQ2UW03/p1668575980848439), we could make use of other settings.